boards/Makefile.include.cortexm_common

```diff
@@ -1,18 +1,34 @@
 # Target triple for the build. Use arm-none-eabi if you are unsure.
 export TARGET_TRIPLE ?= arm-none-eabi
 
-# Toolchain prefix, defaults to target triple followed by a dash, you will most likely not need to touch this.
+# Use TOOLCHAIN environment variable to select the toolchain to use.
+ifeq (,$(TOOLCHAIN))
```

Comment:
I can't think of a better name either, and I have not seen the TOOLCHAIN=arch-prefix- usage yet. Is it widespread?